bootstrap/bootstrapper.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: kubeflow-cluster-admin
+subjects:
+  - kind: ServiceAccount
+    # Reference to upper's `metadata.name`
+    name: default
```

Comment:
Why is this default?

Reply:
This service account is used by k8s client in-cluster-config ( in current namespace).